specification/monitor/resource-manager/Microsoft.Insights/stable/2019-06-01/actionGroups_API.json

```diff
@@ -0,0 +1,879 @@
+                        "$ref": "./examples/getActionGroup.json"
+                    }
+                },
+                "parameters": [
+                    {
+                        "$ref": "#/parameters/ResourceGroupNameParameter"
+                    },
+                    {
+                        "$ref": "#/parameters/ActionGroupNameParameter"
+                    },
+                    {
+                        "$ref": "#/parameters/SubscriptionIdParameter"
+                    },
+                    {
+                        "$ref": "#/parameters/ApiVersionParameter"
+                    }
+                ],
+                "responses": {
+                    "200": {
+                        "description": "The request succeeded.",
+                        "schema": {
+                            "$ref": "#/definitions/ActionGroupResource"
+                        }
+                    },
+                    "default": {
+                        "description": "An error occurred and the action group could not be retrieved. 404: The action group does not exist.",
+                        "schema": {
+                            "$ref": "#/definitions/ErrorResponse"
+                        }
+                    }
+                }
+            },
+            "delete": {
+                "description": "Delete an action group.",
+                "operationId": "ActionGroups_Delete",
+                "x-ms-examples": {
+                    "Delete an action group": {
+                        "$ref": "./examples/deleteActionGroup.json"
+                    }
+                },
+                "parameters": [
+                    {
+                        "$ref": "#/parameters/ResourceGroupNameParameter"
+                    },
+                    {
+                        "$ref": "#/parameters/ActionGroupNameParameter"
+                    },
+                    {
+                        "$ref": "#/parameters/SubscriptionIdParameter"
+                    },
+                    {
+                        "$ref": "#/parameters/ApiVersionParameter"
+                    }
+                ],
+                "responses": {
+                    "200": {
+                        "description": "The action group was successfully deleted."
+                    },
+                    "204": {
+                        "description": "The action group does not exist. It may have already been deleted."
+                    },
+                    "default": {
+                        "description": "An error occurred and the action group could not be deleted.",
+                        "schema": {
+                            "$ref": "#/definitions/ErrorResponse"
+                        }
+                    }
+                }
+            },
+            "patch": {
+                "description": "Updates an existing action group's tags. To update other fields use the CreateOrUpdate method.",
```

Comment:
>CreateOrUpdate [](start = 104, length = 14)

Does the PUT of actionGroup allow partial resource body? In other words, does it act like PATCH?